src/PendingScopedFeatureInteraction.php

```diff
@@ -262,9 +265,12 @@ public function activate($feature, $value = true)
      */
     public function deactivate($feature)
     {
-        Collection::wrap($feature)
+        $features = Collection::wrap($feature)
             ->crossJoin($this->scope())
-            ->each(fn ($bits) => $this->driver->set($bits[0], $bits[1], false));
+            ->map(fn ($bits) => ['feature' => $bits[0], 'scope' => $bits[1], 'value' => false])
+            ->all();
+
+        $this->driver->setAll($features);
```

Comment:
`$this->driver` is our decorator which implements the new contract, so calling `setAll` here is safe.